### ISSUE TITLE
pybugz doesn't finished

### ISSUE THREAD

**Issue by toralf** [williamh/pybugz#104]
Today I realized at my Gentoo Linux Tinderbox a bugz process, which runs since 4 hours. I do wonder if there's any chance to time limit it.

OTOH could consider to use the command "time", right ?

**Comment by asarubbo** [comment]
Yes, it is missing a timeout option, so right now we have a broken bugzilla and pybugz hangs indefinitely
